vnext/local-cli/src/config/configUtils.ts

```diff
@@ -272,11 +273,15 @@ export function importProjectExists(
  * @return The project name.
  */
 export function getProjectName(projectContents: Node): string {
-  return (
+  const name =
     findPropertyValue(projectContents, 'ProjectName') ||
-    findPropertyValue(projectContents, 'AssemblyName') ||
-    ''
-  );
+    findPropertyValue(projectContents, 'AssemblyName');
+
+  if (name === null) {
```

Comment:
Looks like this is actually firing right now for playground-win32 😐. Guessing we haven't kept it up to date with template changes? Will fix that vcproj, and consider adding a better error message if possible.